keps/140-scenario-based-simulation/README.md

```diff
@@ -148,6 +151,25 @@ type ScenarioStatus struct {
   ScenarioResult ScenarioResult
 }
 
+
+type SchedulerStatus string
+const (
+  // SchedulerWillRun indicates the scheduler is expected to start to schedule.
+  // In other words, the scheduler is currently stopped, 
+  // and will start to schedule Pods when the state is SchedulerWillRun.
+  SchedulerWillRun  SchedulerStatus = "WillRun"
```

Comment:
Are `WillRun`, "WillStop" weird name? 😅 
Let me know if there are other good names for them

Reply:
how about introducing two fields that are _simulator expects to_ and _reported from scheduelr_.

```yaml
spec:
  # expected scheduler status by the simulator
  schedulerStatus: "Running|Stopped"
status:
  # reported status from the scheduler
  schedulerStatus: "Running|Stopped"
```

Reply:
That's another idea. Actually I considered that idea when I wrote this, but, it was rejected in my head. 
What I thought about it at that time was:
- If we add an expected scheduler status in ScenarioSpec like what you said, literary that means `expected scheduler status` is part of the spec of "scenario". 
   - It's weird, isn't it? The weirdness comes from the fact that "the expected scheduler's status actually depends on the **status** of the Scenario.".
   - In other words, from the user's point of view, they will never need to touch that field since it's automatically changed by the controller depending on the stage of one `ScenarioStep`.
- I guess if we want to really put it on k8s standard way, we ideally need to create another CRD like `Scheduler` or `SimulationScheduler`. But that's too much. (at least for now.)
    - And, even if we consider creating the `Scheduler` CRD in the future, we can easily migrate the Scenario's `status.SchedulerStatus`; we can keep `status.SchedulerStatus` in that case.

That's why I prefer the current `.status.SchedulerStatus` strategy.